deploy/linux-systemd/tigerbeetle-pre-start

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
I don't think we need any bash specific features here - could we change to `/bin/sh`?

Also, if we rename this as `.sh`, we'll get our shellcheck repo linting to apply to it too.